docs/HOL_Fault-Injection-for-Azure-IaaS-with-OMS/README.md

```diff
@@ -0,0 +1,322 @@
+
+
+## Tasks Overview:
+**1. Setup OMS workspace** This task will show you how to create OMS Workspace using Azure and then find WORKSPACE ID and PRIMARY KEY used to connect to OMS.
+
+**2. Deploy ARM template** In this task you will deploy an ARM template which creates a VMSS of Linux machines and then installs MRP environment on it.
+
+**3. Set up Load Test** In this task you will configure a load test using VSTS.
+
+**4. Inject faults into VMSS** In this task you will learn how to inject faults into VMSS using Azure Portal, PowerShell script or [Chaos-Dingo](https://github.com/jmspring/chaos-dingo).
+
+**5. Analyze load test's results.** In this task you will collect the information related to the load test and fault injection and then analyze it.
+
+
+
+## HOL:
+###Task 1: Setup OMS workspace
+
+**Step 1.** Open Azure and create a new resource group for this HOL.
+
+1. Click on "Resource groups" icon, then on "+ Add".
+
+    ![](media/2.png)
+
+2. Enter a name for your resource group, pick a subscription, select a location and click on "Create".
+
+    ![](media/3.png)
+
+
+**Step 2.** Add "Log Analytics" to this resource group.
+
+1. Open the new resource group and click on "+ Add" button on the "Overview" page to add a new resource.
+
+    ![](media/4.png)  
+
+2. Filter resources by typing "Log Analytics" into the search bar, press "Enter", click "Log Analytics" and click on "Create" button.
+
+    ![](media/5.png)  
+
+3. Create an OMS workspace. Enter name, pick a subscription, location, current resource group and a pricing tier then click on "OK" button.
+
+    ![](media/6.png)  
+    >**Note:** OMS Workspace name has to be unique across Azure.
+
+**Step 3.** Get "Workspace ID" and "Primary Key" from OMS Portal.
+
+1. Navigate to your resource group and click on "Log Analytics" you just created.
+
+2. Click on "OMS Portal".
+
+    ![](media/7.png)  
+
+3. Once OMS portal has loaded, click on "Settings".
+
+    ![](media/8.png)  
+
+4. Click on "Connected Sources", then "Linux Servers" and note the "WORKSPACE ID" as well as the "PRIMARY KEY".
+
+    ![](media/1.png)
+
+**Step 4.** To set up collection of metrics click on "Data", "Linux Performance Counters", "Add the selected performance counters" and then on "Save".
+
+ ![](media/15.png)
+
+
+
+###Task 2: Deploy ARM template
+This task will setup the required environment in Azure.
+
+**Step 1.** Click on "Deploy to Azure" button below.
+
+[<img src="http://azuredeploy.net/deploybutton.png"/>](https://ms.portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2FPartsUnlimitedMRP%2Fmaster%2FHOL_Fault-Injection-for-Azure-IaaS-with-OMS%2Fenv%2Ftemplate%2FlinuxVirtualMachineScaleSet.json)
+[<img src="http://armviz.io/visualizebutton.png"/>](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2FPartsUnlimitedMRP%2Fmaster%2FHOL_Fault-Injection-for-Azure-IaaS-with-OMS%2Fenv%2Ftemplate%2FlinuxVirtualMachineScaleSet.json)
```

Comment:
Can you change the uri so it works within the PR? We can change if to the original repo after we merge it.